localstack-core/localstack/utils/container_utils/container_client.py

```diff
@@ -395,8 +396,14 @@ def to_str(self) -> str:
 
         return ":".join(args)
 
+    def to_dict(self):
```

Comment:
nit: can we add types to this? I feel the name is a bit strange since it returns a `tuple[str, dict[str, str]]` but the name is `to_dict`